net-wireless/uhd/uhd-3.15.0.0-r100.ebuild

```diff
@@ -61,7 +60,7 @@ src_unpack() {
 }
 
```

Comment:
Can't comment on right lines, but:
1. Still no BDEPEND?
2. Why no := on ncurses if linked against it?
3. Why ncurses[tinfo]? It's preferred to not depend on tinfo unless the code truly needs it. Verify it does.
4. Do we need the tinfo patch? Why is there no upstream bug report/PR link (please upstream it if it's needed) or Gentoo bug in the header?

Reply:
If I understand BDEPEND correctly, everything in DEPEND (except the ${RDEPEND} line) should be moved to BDEPEND?  I'll double check the final linkage, gzip might actually be linked in, but I think the rest are just executed.

Reply:
Yeah, so everything we just execute during build => BDEPEND

Linked against is RDEPEND and DEPEND (as we need it at runtime but also at build time to link against & get its headers).

DEPEND-only is usually stuff like header-only libs

Reply:
1. Added
2. added

Reply:
3. Done
4. After some research it looks like the original tinfo patch was from another member of the radio herd a long time ago.  This has now been fixed slightly more properly and I'll report a "final" fix to upstream as well.